data/core/units/elves/Marshal.cfg

```diff
@@ -23,7 +23,7 @@
     {AMLA_DEFAULT}
     cost=54
     usage=fighter
-    description= _ "The longevity and natural intelligence of elves make them apt for military matters, enough even to counter their general distaste for war. Elves remember things much more clearly than humankind, and can often intuit what others can only be trained to do. Certainly, on those rare occasions when an elf sets his mind to war, the strategy that results is often the work of a master."+{SPECIAL_NOTES}+{SPECIAL_NOTES_LEADERSHIP}
+    description= _ "Elves are not often the instigators of open war, but will not shy away when the time comes to do battle. Of course, blindly engaging in combat without any strategy or tactic is foolish, and so the elves maintain a small number of strategically-minded commanders at all times. Selected from the most talented captains, marshals undergo extensive training in warfare and study many varied military styles, including those of the humans and orcs. They are brilliant and unorthodox strategists, and will regularly employ guerilla tactics to great effect in the forests. Engaging elves in their own homes is always a daunting task for any invasion force, and it is precisely because of the skillful maneuvering of the marshals that manage the protection of their people."+{SPECIAL_NOTES}+{SPECIAL_NOTES_LEADERSHIP}
```

Comment:
This seems at odds with your earlier assertion that they are more skilled at small skirmishes.